src/identity/hooks/useName.ts

```diff
@@ -17,7 +17,8 @@ export const useName = (
   queryOptions?: UseNameQueryOptions,
 ) => {
   const { enabled = true, cacheTime } = queryOptions ?? {};
-  const ensActionKey = `ens-name-${address}`;
+  const chainActionKey = chain ? chain.id : 'default-chain';
```

Comment:
I will say let's just have the default value the ETH chain ID. Which I believe is 0, no?

Reply:
it's 1, I'll set the default chain to mainnet like we do for the getName function!